introduction/07-faq.asciidoc

```diff
@@ -0,0 +1,15 @@
+== FAQ
+
+To conclude the Introduction segment of the Learning Path, here are some Frequently Asked Questions people have when embarking on their InnerSource journey.
+
+=== What is the cost/overhead?
```

Comment:
To the host team or the contributing team?
* Host team: 10% effort on your side for 90% effort on contributor side.
* Contributor: 50% more effort to commit.  100% less effort to maintain.

Reply:
> To the host team or the contributing team?

Good point - I meant host team, but perhaps these can be 2 separate questions?

> Host team: 10% effort on your side for 90% effort on contributor side.

In my experience it can be closer to 35/65 e.g.
- If a contribution requires a lot of manual testing - my example would be TV browsers but I believe Android development can also require testing on a multitude of physical devices. Of course, there could be an agreement that the contributing team also performs the testing.
- If a large contribution requires a change in operational support, especially out of hours, up-skilling the host team can take effort. [30 day warranty](https://patterns.innersourcecommons.org/p/30-day-warranty) can of course help with this.
- If a contribution needs a lot of additional work to be production ready, the host team can choose to adopt it, mainly for community health - rejected PRs can discourage further contributions (maybe there's a pattern in there 'Always Accept' or something? I know some OSS communities do this too.).

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/465/commits/b620072deccb16419ba58fc9ec9e03a83c5ded6d